packages/amplify-codegen/src/codegen-config/AmplifyCodeGenConfig.js

```diff
@@ -45,7 +45,9 @@ class AmplifyCodeGenConfig {
     if (!this.constructor.isValidAmplifyProject(project)) {
       return false;
     }
-    const schemaPath = isAbsolute(project.schema) ? relative(this.gqlConfig.configDir, project.schema) : project.schema;
+    const schemaPath = (isAbsolute(project.schema) ? relative(this.gqlConfig.configDir, project.schema) : project.schema)
```

Comment:
This will change the `schemaPath` in `.graphqlconfig.yml` to use posix separators. There will be no functional difference in codegen because node understands paths of both styles.

I think this is the correct behavior because other paths inside `.graphlqconfig.yml` use posix style paths.